src/main/scala/subsystem/RTC.scala

```diff
@@ -22,8 +22,15 @@ trait HasRTCModuleImp extends LazyModuleImp {
   // Use the static period to toggle the RTC
   chisel3.withClockAndReset(pbus.module.clock, pbus.module.reset) {
     val (_, int_rtc_tick) = Counter(true.B, internalPeriod.toInt)
-    outer.clintOpt.foreach { clint =>
-      clint.module.io.rtcTick := int_rtc_tick
+    val isACLINT = p(CLINTKey).get.isACLINT
```

Comment:
This needs to support no-clint designs....

Reply:
I am making modifications